assets/js/pages/AdvisoryDetails/AdvisoryDetailsPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { getAvirosyErrata } from '@lib/api/softwareUpdates';
+import BackButton from '@common/BackButton';
+import AdvisoryDetails from './AdvisoryDetails';
+
+function AdvisoryDetailsPage() {
+  const [advisoryErrata, setAdvisoryErrata] = useState(undefined);
+  const { hostID, advisoryID } = useParams();
+
+  useEffect(() => {
+    try {
+      setAdvisoryErrata(getAvirosyErrata(advisoryID));
```

Comment:
I'm pretty sure you have to `await getAdvisoryData(advisoryID)` before this.

Also, careful with that typo! :fire: